helm/data_template.go

```diff
@@ -375,7 +375,9 @@ func dataTemplateRead(ctx context.Context, d *schema.ResourceData, meta interfac
 		showOnlyAttrValue := showOnlyAttr.([]interface{})
 
 		for _, showFile := range showOnlyAttrValue {
-			showFiles = append(showFiles, showFile.(string))
+			if showFile != nil {
+				showFiles = append(showFiles, showFile.(string))
+			}
```

Comment:
Something you could also do here is use the type assertion to test this, as if it's nil then trying to assert it as a string will fail:

```go

if s, ok := showFile.(string); ok {
  showFiles = append(showFiles, s)
}
```

Reply:
This definitely is a much better way to do it. Thanks for the recommendation!